lib/Fetcher/Client/LegacyGuzzleResponse.php

```diff
@@ -56,8 +56,9 @@ public function getBody() : ? string
     public function getLastModified() : ?\DateTime
     {
         if ($this->response->hasHeader(static::HTTP_LAST_MODIFIED)) {
+            $lastModified->setTimezone(new \DateTimeZone('GMT'));
```

Comment:
```suggestion
```

Not needed here. It's the opposite way. We are consuming the feeds response.

Reply:
Of course, my bad!